src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointAllele.java

```diff
@@ -1,40 +1,57 @@
 package org.broadinstitute.hellbender.tools.spark.sv;
 
+import com.esotericsoftware.kryo.DefaultSerializer;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
 import org.broadinstitute.hellbender.utils.SimpleInterval;
 
+import java.util.ArrayList;
 import java.util.List;
-import java.util.Objects;
 
 import static org.broadinstitute.hellbender.tools.spark.sv.BreakpointAllele.InversionType.INV_3_TO_5;
 import static org.broadinstitute.hellbender.tools.spark.sv.BreakpointAllele.InversionType.INV_5_TO_3;
 
 /**
- * This class represents the allele of an SV breakpoint (a novel adjacency between two genomic locations
+ * This class represents the allele of an SV breakpoint (a novel adjacency between two genomic locations)
  */
+@DefaultSerializer(BreakpointAllele.Serializer.class)
 class BreakpointAllele {
+
     final SimpleInterval leftAlignedLeftBreakpoint;
     final SimpleInterval leftAlignedRightBreakpoint;
     final String insertedSequence;
     final String homology;
-    final InversionType inversionType;
-
-    // not included in equals and hashCode so as not to break grouping by breakpoint allele if the mappings are different
-    final List<String> insertionMappings;
+    private final InversionType inversionType;
 
     public BreakpointAllele(final SimpleInterval leftAlignedLeftBreakpoint, final SimpleInterval leftAlignedRightBreakpoint, final String insertedSequence, final String homology, final boolean fiveToThree, final boolean threeToFive, final List<String> insertionMappings) {
         this.leftAlignedLeftBreakpoint = leftAlignedLeftBreakpoint;
         this.leftAlignedRightBreakpoint = leftAlignedRightBreakpoint;
         this.insertedSequence = insertedSequence;
         this.homology = homology;
         this.inversionType = getInversionType(fiveToThree, threeToFive);
-        this.insertionMappings = insertionMappings;
+    }
+
+    @SuppressWarnings("unchecked")
+    public BreakpointAllele(final Kryo kryo, final Input input) {
+        final String contig1 = input.readString();
+        final int start1 = input.readInt();
+        final int end1 = input.readInt();
+        this.leftAlignedLeftBreakpoint = new SimpleInterval(contig1, start1, end1);
+        final String contig2 = input.readString();
+        final int start2 = input.readInt();
+        final int end2 = input.readInt();
+        this.leftAlignedRightBreakpoint = new SimpleInterval(contig2, start2, end2);
+        this.insertedSequence = input.readString();
+        this.homology = input.readString();
+        this.inversionType = InversionType.valueOf(input.readString());
```

Comment:
Trivial improvement, but you might as well serialize inversionType as an int.  So this line would become:
`        this.inversionType = InversionType.values()[input.readInt()];`